client/app/visualizations/choropleth/index.js

```diff
@@ -219,14 +243,8 @@ function choroplethEditor(ChoroplethPalette) {
         'bottom-right': 'bottom / right',
       };
 
-      $scope.countryCodeTypes = {
-        name: 'Short name',
-        name_long: 'Full name',
-        abbrev: 'Abbreviated name',
-        iso_a2: 'ISO code (2 letters)',
-        iso_a3: 'ISO code (3 letters)',
-        iso_n3: 'ISO code (3 digits)',
-      };
+      $scope.countryCodeTypes = {};
+
 
       $scope.templateHint = `
```

Comment:
You should fix this template as well (depending on map type). Also you probably need to update `inferCountryCodeType()` function (not sure - can you check please?)